examples/java/src/main/java/org/apache/beam/examples/cookbook/TriggerExample.java

```diff
@@ -121,7 +121,9 @@
  * <p>The example will try to cancel the pipelines on the signal to terminate the process (CTRL-C)
  * and then exits.
  */
-@SuppressWarnings("nullness") // TODO(https://issues.apache.org/jira/browse/BEAM-10402)
+@SuppressWarnings({
+  "nullness" // TODO(https://issues.apache.org/jira/browse/BEAM-10402)
```

Comment:
There are a number of changes like this that I did just to make it more sane to automatically add the rawtype warnings. I can revert if you like or separate into another commit.

Reply:
SG. It's a mechanical change so no problem leaving it in this commit.